examples/pytorch_mnist/pytorch_mnist_demo.ipynb

```diff
@@ -221,7 +221,6 @@
     "\n",
     "    # K-fold Cross Validation model evaluation\n",
     "    for fold, (train_ids, test_ids) in enumerate(kfold.split(dataset)):\n",
-    "\n",
```

Comment:
These don't look like formatter changes?

Reply:
that is from `black[jupyter]`